src/NCronJob/Configuration/Builder/NCronJobOptionBuilder.cs

```diff
@@ -190,13 +195,16 @@ internal class StartupStage<TJob> : IStartupStage<TJob> where TJob : class, IJob
     private readonly ConcurrencySettings settings;
     private readonly JobRegistry jobRegistry;
     private readonly JobOptionBuilder jobOptionBuilder;
+    private readonly ICollection<JobDefinition> jobDefinitions;
```

Comment:
nitpick: I would like to keep it a `IReadOnlyCollection` so that the API is telling that this collection isn't going to change somewhere.

Reply:
Fixed